src/models/entity/summary.js

```diff
@@ -0,0 +1,21 @@
+// Entity summary
+const DATASOURCES = {
+  NCBIGENE: 'http://identifiers.org/ncbigene/',
+  HGNC: 'http://identifiers.org/hgnc/',
+  UNIPROT: 'http://identifiers.org/uniprot/',
+  GENECARDS: 'http://identifiers.org/genecards/'
+};
+
+class EntitySummary {
+  constructor( dataSource, displayName, localID, description, aliases, aliasIds, xref ){
```

Comment:
There are so many args, it's hard to know what is what.

It's better to do this:

```js
constructor({ dataSource, displayName, localID, description, aliases, aliasIds, xref }){
  // ...
}
```

That is, you pass an object whose fields are the args.  That way you can see what arg is what based on the object keys.

Reply:
Or just do...

```js
constructor(options){
  _.assign(this, DEFAULTS, options);
}
```

... where you define `DEFAULTS` in the top scope.

Reply:
ok good call. much better...